nodewatcher/core/frontend/urls.py

```diff
@@ -12,4 +14,6 @@
     # Frontend components auto-discovery
     components.pool.discover_components()
 
-    urlpatterns = components.pool.get_urls()
+    urlpatterns = components.pool.get_urls() + [
+        urls.url(r'^api/v2/', urls.include(api.router.urls)),
```

Comment:
Because to me it is not part of frontend. :-) API can be used by mobile apps, for example. While our front-end is our "web frontend" in fact.